web/src/main/java/com/futurewei/alcor/web/restclient/GatewayManagerRestClient.java

```diff
@@ -25,8 +29,20 @@
     public ZetaGatewayIpJson createVPCInZetaGateway(Object args) throws Exception {
         String url = zetaManagerUrl + "/vpcs";
         VpcInfoSub vpcInfoSub = (VpcInfoSub) args;
-        HttpEntity<VpcInfoSub> request = new HttpEntity<>(vpcInfoSub);
-        return restTemplate.postForObject(url, request, ZetaGatewayIpJson.class);
+        ObjectMapper Obj = new ObjectMapper();
+        String jsonStr = Obj.writeValueAsString(vpcInfoSub);
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        HttpEntity<String> request = new HttpEntity<>(jsonStr, headers);
+        Object response = restTemplate.postForObject(url, request, Object.class);
+        //return restTemplate.postForObject(url, request, ZetaGatewayIpJson.class);
```

Comment:
We can remove the codes that've been commented out.

Reply:
comment removed.